chat-plugins/wisp.js

```diff
@@ -1095,6 +1095,50 @@ exports.commands = {
 		"/background set [user], [image] - Sets a users profile background.",
 		"/background delete [user] - Deletes a users profile background.",
 	],
+	music: {
+		set: function (target, room, user) {
+			if (!this.can('music')) return false;
```

Comment:
This isn't how we just add a permission.

You need to also add it to ``config/config-example.js``